app/views/shared/_form_errors.html.erb

```diff
@@ -0,0 +1,11 @@
+<% if model.errors.any? %>
+<div id="error_explanation">
+  <h2><%= pluralize(model.errors.count, "error") %> prohibited this <%= model.class.name %> from being saved:</h2>
```

Comment:
What would you think of adding a new translation for this string?

Note: please use `model.class.model_name.human`.

Reply:
I replace `model.class.name` on https://github.com/tomatoes-app/tomatoes/pull/299/commits/c3424ced55d655cf34bbbe4825597c7e42a11286.